content/josefina.md

```diff
@@ -26,7 +26,7 @@ I also worked on mapping the contigs from two different assemblies from the same
 
 The two assemblies are:     
 (1) combined_transcripts_cleaned_fasta_cap_contigs.fasta (assembled using The Eel-pond mRNA-seq Protocol)     
-(2) Hglaberrima_RNC_transcriptome_Contigs.fasta.zip (assembled by Vladimir Mashanov, DOI: 10.1186/1471-2164-15-357)     
+(2) Hglaberrima_RNC_transcriptome_Contigs.fasta.zip (assembled by Vladimir Mashanov, dx.doi.org/10.1186/1471-2164-15-357)     
```

Comment:
If you want it to be a link you have to put it in '<>' and add '//' or 'https://' to the front.

'<https://dx.doi.org/10.1186/1471-2164-15-357>'